pkg/router/template/router.go

```diff
@@ -553,6 +555,10 @@ func (r *templateRouter) shouldWriteCerts(cfg *ServiceAliasConfig) bool {
 		if hasRequiredEdgeCerts(cfg) {
 			return true
 		} else {
+			if cfg.TLSTermination == routeapi.TLSTerminationReencrypt && hasReencryptDestinationCACert(cfg) {
+				return true
+			}
+
 			msg := fmt.Sprintf("a %s terminated route with host %s does not have the required certificates.  The route will still be created but no certificates will be written",
```

Comment:
So how about I just add a specific info level message if the above `if termination is re-encrypt` block saying something like:
`a reencrypt route with host %s does not have an edge certificate, using default router certificate`?   The message below is still valid for edge terminated routes.